collect_app/src/main/java/org/odk/collect/android/activities/ViewOnlyFormHierarchyActivity.java

```diff
@@ -26,6 +29,19 @@
  * features that allow the user to edit the form instance.
  */
 public class ViewOnlyFormHierarchyActivity extends FormHierarchyActivity {
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        onBackPressedCallback.remove();
+        getOnBackPressedDispatcher().addCallback(new OnBackPressedCallback(true) {
```

Comment:
Is this not just the same as not adding a callback as it just calls `finish()`?

Reply:
Right, fixed.